lambda/simple-hello-service/readme.md

```diff
@@ -0,0 +1,34 @@
+# Simple hello service
+A container based codepipes lambda function deployments
```

Comment:
This sentence doesn't make sense?
"A container based lambda function deployment for Code Pipes"?